src/components/BlogDetails/BlogDetailsComponents/BlogCommentForm.jsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import React, { useState } from "react";
+
+const BlogCommentForm = () => {
+  const [form, setForm] = useState({ name: "", email: "", comment: "" });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setForm({ ...form, [name]: value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("Form submitted:", form);
+    setForm({ name: "", email: "", comment: "" });
```

Comment:
Please use try{}catch{} and write the logic and after that set the form data like this: setForm({})